Add CUDA Enhanced Compatibility & `devel` Container Notices (#233)
rapidsai/docs: _notices/rdn0003.md

---
layout: notice
parent: RAPIDS Developer Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rdn
# Update meta-data for notice
notice_id: 3 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Devel Container Release Change"
notice_author: RAPIDS Ops
notice_status: Completed

> notice_status_color: green

Reviewer: should this be yellow since 21.12 is in the process of being released and the nightly packages for CUDA 11.5 are not currently available?

Author: Good catch. I think since we've stopped building the `21.12` non-`11.5` nightlies at this point, it's probably fine to leave it as `green`/`Completed`.

# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Release Change
notice_rapids_version: "v21.12"
notice_created: 2021-11-18
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2021-11-18
---

## Overview

> RAPIDS `devel` containers will only support CUDA `11.5` effective with the `21.12` release.

Reviewer 2: ```suggestion
Effective with the `21.12` release, RAPIDS `devel` packages will only be distributed for CUDA `11.5`.
```
